marge/app.py

```diff
@@ -154,6 +160,12 @@ def main(args=sys.argv[1:]):
     with _secret_auth_token_and_ssh_key(options) as (auth_token, ssh_key_file):
         api = gitlab.Api(options.gitlab_url, auth_token)
         user = user_module.User.myself(api)
+        if options.max_ci_time_in_minutes:
+            logging.warning(
+                "--max-ci-time-in-minutes is DEPRECATED, use --ci-timeout %dmin",
+                options.max_ci_time_in_minutes
+            )
+            options.ci_timeout = timedelta(minutes=options.max_ci_time_in_minutes)
```

Comment:
@aschmolck I rebased my branch with unit tests for the `app` on top of this and it caught the error here! It should be:
```python
 options.ci_timeout = timedelta(minutes=options.max_ci_time_in_minutes).total_seconds()
```

Don't make a release until I push that branch with the fix...